### DISCUSSION TITLE
Github Copilot available in pycharm

### DISCUSSION THREAD

**Discussion by SaikatMandal61299** [community/community#7131] [Copilot Conversations] [answered]
I tried to install Github Copilot in pycharm but its's not available in the plugins store.

**Comment by davecheney** [comment, marked as answer]
@SaikatMandal61299 we require Pycharm 2021.2. Which version are you using?

**Reply by ocarlisle** [reply]
Do you know how to log into copilot on a Mac with PyCharm? the outline in the website only shows instructions for a windows machine and the Mac machine doesn't have a tools option unless you go into preferences and when I navigate there, there is no option to log into my GitHub account

**Reply by jansorg** [reply]
@ocarlisle Apologies for the delay. A missing tools menus is unexpected, but alternatively you can login via `Help > Find Action` and entering `copilot: login` to locate the action. Using a `copilot:` as prefix will show the other available GitHub Copilot Actions

**Comment by SaikatMandal61299** [comment]
I have 2021.1
I will update it.
Thanks for your help 👍

**Reply by Guitarhead101** [reply]
I have 2021.2 and likewise it wont install for me?
